src/main/java/com/o19s/es/ltr/feature/store/StoredFeature.java

```diff
@@ -111,7 +110,7 @@ public StoredFeature(String name, List<String> params, String templateLanguage,
     }
 
     public StoredFeature(String name, List<String> params, String templateLanguage, XContentBuilder template) {
-        this(name, params, templateLanguage, Strings.toString(Objects.requireNonNull(template)), false);
+        this(name, params, templateLanguage, Objects.requireNonNull(template).toString(), false);
```

Comment:
ooh. this is much more standardized than the hackery I was working on previously. nice!